utility.go

```diff
@@ -0,0 +1,19 @@
+package testcontainers
+
+import (
+	"math/rand"
+	"time"
+)
+
+var alfabet = []rune("123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz")
+
+// RandomString returns a random string
+func RandomString(length int) string {
```

Comment:
would it be better to generate a UUID? rather than write our own `RandomString` function here? if it is just being used as an identifier, we just need to guarantee its uniqueness, no?

Reply:
Totally agree, and I'm using it only for tests, which makes no sense to maintain new code.

Thanks for the feedback!